platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/snapshot/SignedStateFileReader.java

```diff
@@ -72,50 +65,36 @@ public static List<SavedStateInfo> getSavedStateFiles(
                 .getSavedStateFiles(mainClassName, platformId, swirldName);
     }
 
-    /**
-     * This is a helper class to hold the data read from a state file.
-     * @param state the Merkle tree state
-     * @param hash the hash of the state
-     * @param sigSet the signature set
-     * @param fileVersion the version of the file
-     */
-    public record StateFileData(
-            @NonNull MerkleRoot state, @NonNull Hash hash, @Nullable SigSet sigSet, int fileVersion) {}
-
     /**
      * Reads a SignedState from disk using the provided snapshot reader function. If the reader throws
```

Comment:
There's not a reader function provided anymore, too. Please update the javadoc accordingly.

Reply:
Fixed

Reply:
I don't believe this particular one has been fixed yet.

Reply:
Now it is.